database/sql/V065__auction_orders.sql

```diff
@@ -0,0 +1,7 @@
+-- All orders that were part of an Auction. An order can appear in multiple auctions.
+CREATE TABLE auction_orders (
+  auction_id bigint NOT NULL,
+  order_uid bytea NOT NULL,
```

Comment:
```suggestion
  order_uids bytea[] NOT NULL
```
Are we going to query the table by order id? If not, this can be optimized.

Reply:
updated